test/webdriverio/customCommands.js

```diff
@@ -21,4 +30,102 @@ browser.addCommand("login", function (serviceName) {
         return new Error('Cannot login to ' + serviceName);
       }
     });
-}); 
\ No newline at end of file
+});
+
+browser.addCommand("testIsVisible", function (selector) {
```

Comment:
Commands testIsVisible, testIsNotVisible, testIsExisting, testIsNotExisting save only two lines of code, but make tests more obscure.

Lets use the the framework's code (sample below):

```js
browser
    .isVisible('#someRandomVisibleElement').then(function(isVisible) {
        expect(isVisible).toBeTruthy();
    })
```